dev_docs/docs/Fundamentals/Logging/overview.md

```diff
@@ -0,0 +1,90 @@
+# Overview
+
+VC Platform supports two logging libraries out of the box:
+
+* Serilog (built-in in the platform). 
+
+    [Read more](https://serilog.net/){ .md-button }
+
+* MS Azure Application Insights (as separated module). 
+
+    [Read more](https://learn.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview){ .md-button }
+
+!!! NOTE
+
```

Comment:
Add link to how to update article

Reply:
![image](https://github.com/VirtoCommerce/vc-docs/assets/106367190/685f939d-f486-45cb-8fc8-5a7602b8894b)
There's one in the references list